src/core/SampleRecordHandle.cpp

```diff
@@ -30,7 +30,7 @@
 #include "Pattern.h"
 #include "SampleBuffer.h"
 #include "SampleTrack.h"
-
+#include "debug.h"
```

Comment:
As the assert macro in `debug.h` doesn't do anything if `LMMS_DEBUG` is not defined, the '#ifdef' check in line 119 can be removed.